src/coreclr/ToolBox/superpmi/superpmi-shared/compileresult.cpp

```diff
@@ -1083,3 +1049,8 @@ bool CompileResult::fndRecordCallSiteMethodHandle(ULONG instrOffset, CORINFO_MET
     }
     return false;
 }
+
+void CompileResult::dmpCrSigInstHandleMap(DWORD key, DWORDLONG value)
```

Comment:
who uses this function?

Reply:
It's automatically called by the dump routines, generated because the map is listed in crlwmlist.h